docs/reference/streamfield/index.md

```diff
@@ -0,0 +1,11 @@
+(StreamField_reference)=
+
+# StreamField reference
+---
+```{toctree}
+---
+maxdepth: 2
+---
+blocks
+widget_api
+```
```

Comment:
this is malformed and is missing content in the original index file - see below (note you will have to view it raw). or see `docs/topics/index.md` for a correct example.

See the documentation for how to do this also https://docs.wagtail.org/en/latest/contributing/documentation_guidelines.html#tables-of-contents


```
    ```{toctree}
    ---
    maxdepth: 2
    ---
    blocks
    widget_api
    ```
```